packaging/linux/rpm/package_binaries.sh

```diff
@@ -111,12 +111,12 @@ export debian_arch=i386
 # On Fedora, it would be more correct to require "libXScrnSaver", which
 # provides libXss.so. Unfortunately that doesn't work on OpenSUSE. This is the
 # most compatible set of dependencies we've found.
-dependencies="Requires: at, fuse, libXss.so.1"
+dependencies="Requires: at, fuse, libXss.so.1, initscripts"
```

Comment:
This is needed for `killall`, used by `run_keybase`.

Reply:
Could you mention that in the comment above?